DFC.ServiceTaxonomy.ApiFunction/CypherQueriesNotes/GetSkillsGapForOccupationAndGivenSkills.md

```diff
@@ -0,0 +1,163 @@
+﻿
+# GetOccupationsWithMatchingSkillsDetailed Query Notes
+
+## Current Query
+
+Here's the current query, formatted and with params replaced by literals...
+
+```
+ MATCH (o:esco__Occupation ) <-[r:esco__isEssentialSkillFor|esco__isOptionalSkillFor]- (s:esco__Skill ) -[:skos__broader]->(d) where d.skos__notation starts with 'S' and o.uri = 'http://data.europa.eu/esco/occupation/9581c837-fee7-4068-9618-1116fd93d723' 
```

Comment:
`skos__broader` will only match skills that have a 1st degree `skos__broader` skill category.
Should it be `skos__broaderTransitive`?

Reply:
there is a load of versions in there. look at entry 4. for query in use